tms/models/res_config_settings.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    # Groups
+
+    group_tms_team = fields.Boolean(
+        string="Manage Teams", implied_group="tms.group_tms_team"
+    )
+    group_tms_crew = fields.Boolean(
+        string="Manage Crews", implied_group="tms.group_tms_crew"
+    )
+    group_tms_driver_license = fields.Boolean(
+        string="Manage Driver License", implied_group="tms.group_tms_driver_license"
+    )
+
+    group_tms_vehicle_insurance = fields.Boolean(
+        string="Manage Vehicle Insurance",
+        implied_group="tms.group_tms_vehicle_insurance",
+    )
+
+    group_tms_route = fields.Boolean(
+        string="Manage Routes", implied_group="tms.group_tms_route"
+    )
+
+    group_tms_route_stop = fields.Boolean(
+        string="Manage Route Stops", implied_group="tms.group_tms_route_stop"
+    )
+
+    group_tms_uom = fields.Boolean(
+        string="Units of Measure", implied_group="tms.group_tms_uom"
+    )
+
+    # Custom Fields
+
+    driver_license_security_days = fields.Integer(
+        required=True,
+        string="Driver license security days",
+        config_parameter="tms.default_driver_license_security_days",
+    )
+
+    vehicle_insurance_security_days = fields.Integer(
+        required=True,
+        string="Vehicle Insurance security days",
+        config_parameter="tms.default_vehicle_insurance_security_days",
+    )
+
+    tms_length_uom = fields.Many2one(
+        "uom.uom",
+        domain="[('category_id', '=', 'Length / Distance')]",
```

Comment:
The domain based on the title can be problematic, especially due to translation. Even if it were based on a reference, we could still have issues since categories can be deleted. I think it would be better to leave the unit of measure selection fields without a domain. What do you think?

Reply:
If necessary, we could typify the category of the category. For example, we could create a field called tms_uom_category of type selection and then assign the tms category to the unit of measure category. However, since this is essentially a one-time configuration, if there is no need for verification in other parts of the code, I would leave it open and assign the responsibility for proper definition to the user/consultant.